libs/datasets/demographics.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional, List, NewType, Dict
+import dataclasses
+
+import pandas as pd
+
+
+Distribution = NewType("Distribution", Dict[str, float])
```

Comment:
I think the str is just a bucket name, not DistributionBucket. Add comment to clarify.